bindings/kafka/metadata.yaml

```
@@ -268,4 +268,31 @@ metadata:
       This is potentially insecure and not recommended for use in production.
     example: "true"
     default: "false"
-    type: bool
\ No newline at end of file
+    type: bool
+  - name: schemaRegistryURL
+    type: string
+    description: |
+      The Schema Registry URL.
+    example: '"http://localhost:8081"'
+  - name: schemaRegistryAPIKey
+    type: string
+    description: |
+      The Schema Registry credentials API Key.
+    example: '"XYAXXAZ"'
+  - name: schemaRegistryAPISecret
+    type: string
+    description: |
+      The Schema Registry credentials API Secret.
+    example: '"ABCDEFGMEADFF"'
+  - name: schemaCachingEnabled
+    type: bool
+    description: |
+      Enables caching for schemas.
+    example: '"true"'
+    default: '"true"'
+  - name: LatestSchemaCacheTTL
+    type: duration
+    description: |
+      The TTL for schema caching when publishing a message with latest schema available.
+    example: '"5min"'
+    default: '"5min"'
```

Incorrect format for Go durations:
```suggestion
      example: '"5m"'
      default: '"5m"'
```